.github/workflows/dispatch-build-bottle.yml

```
@@ -145,19 +145,19 @@ jobs:
         env:
           HOMEBREW_BINTRAY_USER: brewtestbot
           HOMEBREW_BINTRAY_KEY: ${{secrets.HOMEBREW_BINTRAY_KEY}}
-          HOMEBREW_GPG_PASSPHRASE: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY_PASSPHRASE }}
-          BREWTESTBOT_NAME_EMAIL: "BrewTestBot <1589480+BrewTestBot@users.noreply.github.com>"
         run: |
           cd ~/bottles
-          brew pr-upload --verbose --keep-old --committer="$BREWTESTBOT_NAME_EMAIL"
+          brew pr-upload --verbose --keep-old --no-commit --root-url="https://ghcr.io/v2/homebrew/core"
```

```suggestion
          brew pr-upload --verbose --keep-old --root-url="https://ghcr.io/v2/homebrew/core" --committer="$BREWTESTBOT_NAME_EMAIL"
```

Best to move `--no-commit` to Bintray so it's easier to remove.

---

Is that not what I did? This is the part where things are uploaded to Bintray.

---

`--root-url="https://ghcr.io/v2/homebrew` suggests you are uploading to GitHub packages.

---

Hmm, ok, I think I see where you're going.

---

Hmmm, ok, gotcha